packages/block-library/src/post-hierarchical-terms/variations.js

```diff
@@ -7,11 +7,11 @@ const variations = [
 	{
 		name: 'category',
 		title: __( 'Post Categories' ),
-		description: __("Display a post's categories."),
+		description: __( "Display a post's categories." ),
```

Comment:
While we're here, should we change this to something less fiddly, such as "Display the categories of a post."?

Reply:
I think we can land now and handle this in combination with an icon change, what do you think?

Reply:
Sure.